src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -33,6 +31,7 @@ internal class ApplicationInsightsLogger : ILogger
         internal const string MetricMinKey = "min";
         internal const string MetricMaxKey = "max";
         internal const string MetricStandardDeviationKey = "standarddeviation";
+        internal const string MessageEnqueuedTime = "EnqueuedTime";
```

Comment:
Should this be bumped up to `LogConstants` so other triggers can take advantage?